src/sentry/snuba/metrics/fields/base.py

```diff
@@ -219,6 +219,14 @@ def _get_entity_of_metric_mri(
         entity_keys_set = frozenset(
             {EntityKey.MetricsCounters, EntityKey.MetricsSets, EntityKey.MetricsDistributions}
         )
+    elif use_case_id is UseCaseID.ESCALATING_ISSUES:
+        entity_keys_set = frozenset(
+            {
+                EntityKey.GenericMetricsCounters,
+                EntityKey.GenericMetricsSets,
+                EntityKey.GenericMetricsDistributions,
```

Comment:
Since right now you only support `c` for counters, are you sure you want to run 3 queries to check the entity key? I would keep `GenericMetricsCounters` for now and then extend it if you were to add additional entity types to the `ErrorMRI`.